packages/microcode/microcode.spec

```diff
@@ -26,11 +26,22 @@ Source3: https://github.com/intel/Intel-Linux-Processor-Microcode-Data-Files/arc
 
 # Lets us install "microcode" to pull in the AMD and Intel updates.
 Requires: %{_cross_os}microcode-amd
-Requires: %{_cross_os}microcode-intel
+Requires: %{_cross_os}microcode-intel-other
+Requires: %{_cross_os}microcode-intel-ec2
 
 %description
 %{summary}.
 
+%package ec2
+Summary: Microcode for EC2 processors
+URL: https://github.com/bottlerocket-os/bottlerocket/tree/develop/packages/microcode
```

Comment:
This URL is no longer correct and since we're touching the spec we should take the opportunity to fix it.